ArmaReforgerServerTool/Forms/Main.cs

```diff
@@ -21,6 +22,9 @@ public Main()
         {
             InitializeComponent();
 
+            // Set the window title with the version number
+            this.Text = $"ReforgerServerApp - {GetAppVersion()}";
```

Comment:
Should be "Arma Reforger Dedicated Server Tool", There are some messagebox constants in `Utils/Constants.cs`, instead of having:
```
. . . 
public static string WARN_MESSAGEBOX_TITLE_STR = "Arma Reforger Dedicated Server Tool - Warning";
public static string INFO_MESSAGEBOX_TITLE_STR = "Arma Reforger Dedicated Server Tool - Information";
. . .
```

we could have `public static string APPLICATION_NAME = "Arma Reforger Dedicated Server Tool";` and each of the messagebox strings can be `APPLICATION_NAME + blah` and then you could refer to `APPLICATION_NAME` here to keep things consistent across the app